### PR TITLE
Ranger Hive client installation

### PR DESCRIPTION
**Pull request by akaranjkar-qu**

### CODE REVIEW

File: hive/ranger-client.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+source /usr/lib/hustler/bin/qubole-bash-lib.sh
+
+# Import function to restart HS2
+source /usr/lib/qubole/bootstrap-functions/hive/hiveserver2.sh
```
Comment (akaranjkar-qu) [inline hive/ranger-client.sh:4]: Provided by https://github.com/qubole/bootstrap-functions/pull/4

**Comment by akaranjkar-qu** [comment]
Depends on HS2 restart functions provided by https://github.com/qubole/bootstrap-functions/pull/4

File: hive/ranger-client.sh [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+# -p: Ranger admin port. Defaults to `6080`
+# -r: Ranger repository name. Defaults to `hivedev`
+# -s: Path to SQL connector jar. Defaults to jar for MySQL 5.1.34
+#     `/usr/lib/hive1.2/lib/mysql-connector-java-5.1.34.jar`
+#
+function install_ranger() {
+    HOST=localhost
+    PORT=6080
+    REPOSITORY_NAME=hivedev
+    SQL_CONNECTOR_JAR=/usr/lib/hive1.2/lib/mysql-connector-java-5.1.34.jar
```
Comment (nitinik) [inline hive/ranger-client.sh]: @himanshu This should be version independent . Can we use wildcard ?

**Comment by himanshu-mishra** [comment]
Ranger needs to be installed only on machines where HS2 will run.
[is_hs2_configured()](https://github.com/qubole/bootstrap-functions/pull/4/files#diff-789fb9184651d65deb34eaa6eb9a64e3R9) can tell whether HS2 will be running on current machine or not.
Should this be handled in `ranger-client.sh` or in the final bootstrap the user writes ? @nitinik

**Review by nitinik — COMMENTED** [review]
> Ranger needs to be installed only on machines where HS2 will run.
> [is_hs2_configured()](https://github.com/qubole/bootstrap-functions/pull/4/files#diff-789fb9184651d65deb34eaa6eb9a64e3R9) can tell whether HS2 will be running on current machine or not.
> Should this be handled in `ranger-client.sh` or in the final bootstrap the user writes ? @nitinik

+1 cc @akaranjkar-qu

File: hive/ranger-client.sh (hunk 2 above)
Comment (himanshu-mishra) [reply hive/ranger-client.sh]: This code has been removed. We do not need to configure SQL Connector.

**Review by himanshu-mishra — APPROVED** [review]
LGTM

**Pull request merged**